rosdep/python.yaml

```diff
@@ -5087,6 +5087,9 @@ python3-bitstring:
     '*': [python3-bitstring]
     '7': [python36-bitstring]
   ubuntu: [python3-bitstring]
+python3-bleak:
```

Comment:
Hi @Geibinger, you can see below that our automated checks picked up that `python3-bleak` is not available on `jammy` or `focal`. You might have to do something like this to signal that it is not available:

```yaml
python3-bleak:
  debian: [python3-bleak]
  ubuntu:
    '*': [python3-bleak]
    focal: null
    jammy: null
```